cmake/spheral/SpheralAddLibs.cmake

```diff
@@ -33,11 +33,9 @@ function(spheral_add_obj_library
   blt_add_library(NAME        Spheral_${package_name}
                   HEADERS     ${${package_name}_headers}
                   SOURCES     ${${package_name}_sources}
-                  DEPENDS_ON  ${spheral_depends} ${spheral_blt_depends} ${spheral_blt_cxx_depends} ${${package_name}_ADDITIONAL_DEPENDS} ${SPHERAL_CXX_DEPENDS}
+                  DEPENDS_ON  ${spheral_blt_depends} ${spheral_blt_cxx_depends} ${${package_name}_ADDITIONAL_DEPENDS} ${SPHERAL_CXX_DEPENDS}
                   OBJECT TRUE
                   )
-  target_include_directories(Spheral_${package_name} PRIVATE ${SPHERAL_INCLUDES})
-  target_include_directories(Spheral_${package_name} SYSTEM PRIVATE ${SPHERAL_EXTERN_INCLUDES})
```

Comment:
@jmikeowen Oops, need to add this one back in on your branch otherwise Spheral doesn't pick up the polyclipper includes anymore